libs/deso-protocol/src/lib/identity/Identity.ts

```diff
@@ -339,7 +343,7 @@ export class Identity {
   ) {
     // don't submit the transaction, instead just return the api response from the
     // previous call
-    if (options?.broadcast === false) return;
+    if (options?.broadcast === false) return { TransactionHex };
     // server app? then you can't call the iframe
     if (this.host === 'server') throw Error(SERVER_ERROR);
```

Comment:
should we update this function to handle signing w/o identity? basically allows us to pass in options in `RequestOptions` so any function that calls `submitTransaction` can get support for signing w/o identity. that would also allow us to support `submitTransaction` when `this.host === 'server'` and we have the signing w/o identity params.

Reply:
Yes, but I'll knock that out in a different PR

Reply:
okay that works for me.